backend/src/polling_station/structs.rs

```diff
@@ -52,6 +52,7 @@ pub struct PollingStationStatusEntry {
 pub enum PollingStationStatus {
     FirstEntry,
     FirstEntryInProgress,
+    FirstEntryUnfinished,
```

Comment:
As discussed on Mattermost, a comment with explanation about the difference between the statuses will be added here

Reply:
Done in 56f56f04a185522c619255adb566a6e3bd92ee24 :)